app/api/permissions/specs/fixtures.js

```diff
@@ -0,0 +1,126 @@
+import db from 'api/utils/testing_db';
```

Comment:
pls make sure you write the fixtures as ts also.
you can use the DBFixtures type.
https://github.com/huridocs/uwazi/blob/042696c744116b6e1cc52385d997c26f2dce3fde/app/api/utils/testing_db.ts#L20-L25

pls extend the type with the new types you create if needed.
also the default export from testing db is deprecated use ```import { testingDB } ``` instead.

Reply:
done